secureconfig/secureconfigparser.py

```diff
@@ -81,18 +79,18 @@ def get(self, sec, key, default=None):
         return val
 
     def set(self, sec, key, new_val, encrypt=False):
-        '''If the value should be secured, encrypt and update it; 
+        '''If the value should be secured, encrypt and update it;
             Otherwise just update it.  supply encrypt=True to encrypt
             a value that was not previously encrypted.
         '''
         if not self.has_option(sec, key):
-            if encrypt==True:
-                new_val = self.ck.sigil + self.ck.encrypt(new_val)
+            if encrypt is True:
+                new_val = self.ck.sigil + self.ck.encrypt(new_val.encode()).decode()
```

Comment:
matches usage below -- assumes `new_val`, which is part of the external API, is a string, but `ck.encrypt` both receives and returns bytes.